source/ref_trainings.rst

```diff
@@ -393,3 +400,53 @@ These names must be `head_input.pkl` and `head_output.pkl`, respectively.
     mlflow.log_artifact('head_input.pkl', 'model')
     train_y.head().to_pickle('head_output.pkl')
     mlflow.log_artifact('head_output.pkl', 'model')
+
+
+********************************************
+MLFlow Project toolchain
+********************************************
+
+MLFlow Project toolchain is a lightweight version of :ref:`MLFlow toolchain`.
+
+The main difference is that MLFlow Project toolchain does not require user stores models using
+`MLFlow Tracking API <https://www.mlflow.org/docs/latest/tracking.html>`_
+and therefore does not require models stored in MLFlow format as a resulted artifact.
+
+Instead, MLFlow Project toolchain relies only on
+`MLFlow Project functionality <https://www.mlflow.org/docs/latest/projects.html>`_  to run training script
+and manage dependencies. User can store result artifacts in any format as he wishes.
```

Comment:
```suggestion
and manage dependencies. User can store result artifacts in any format as they wish.
```

Reply:
I'd suggest totally remove `as he wish` and leave it as `User can store result artifacts in any format`. BTW: what does `any format` means? Maybe we should count them?

Reply:
There is no specific set of formats. Just any files